websites/T/timolia/presence.ts

```diff
@@ -0,0 +1,73 @@
+const presence = new Presence({
+    clientId: "872421983554502717"
+  }),
+
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "large_image",
+    details: document.location.href,
```

Comment:
details to href????

Reply:
href displays the current website url

Reply:
*But why use it on details*

Reply:
If you are on a website where there is no explicit description only the URL is shown

Reply:
Then something like "Unsupported Page" or just nothing is better in my opinion :/

Reply:
alright, then i remove it :)